pytest_postgresql/factories/_pg.py

```
@@ -0,0 +1,75 @@
+"""Internal module for functions handling pg_ctl discovery."""
+
+import logging
+import os
+import platform
+import subprocess
+from typing import Iterable
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)